src/main.rs

```diff
@@ -47,6 +47,8 @@ use libtock_drivers::timer::Timer;
 use libtock_drivers::timer::Timestamp;
 use libtock_drivers::usb_ctap_hid;
 
+libtock_core::stack_size! {0x4000}
```

Comment:
This is a bigger stack than before, or are units different?

Reply:
Nope, exactly the same as currently stated in `deploy.py` AFAICT

Reply:
Hmm, I might have confused it with the example stack size from `layout.ld`.